src/Commands/utilities/commands.js

```diff
@@ -20,6 +20,11 @@ module.exports = {
 	 */
 	async execute(interaction, client) {
 
+		// technically, this can be moved into the ready event as well
+		if (!interaction.client.application.commands.cache.size) {
+			await interaction.client.application.commands.fetch();
+		}

```

Comment:
As mentioned in the comment, you could possibly even move the fetch to your ready event